web_teleop/launch/app.launch

```diff
@@ -0,0 +1,5 @@
+<launch>
+  <include file="$(find uw_gazebo)/launch/house_simulation.launch" />
```

Comment:
This was probably for testing, right?

Reply:
Yes, this should be what I added when testing. I'll remove it.